docs/guide/blog.md

```diff
@@ -413,11 +228,11 @@ We've implemented all the code necessary to create new posts and store them on c
 Let's try it out! If the chain is yet not started, run `starport chain serve`. Create a post:
 
 ```
-blogd tx blog createPost foo bar --from alice
+blogd tx blog createPost foo bar --from cosmonaut
```

Comment:
```suggestion
blogd tx blog create-post foo bar --from alice
```

`cosmonaut` is by default not in the config.yml file and we did not mention to add it to the user. Should we stick with alice for simplicity?

Reply:
Absolutely. Not sure a cosmonaut sneaked in.